it/v1/endpoints/RetrievePensionsChargesControllerISpec.scala

```diff
@@ -134,7 +163,12 @@ class RetrievePensionsChargesControllerISpec extends IntegrationBaseSpec {
           (Status.INTERNAL_SERVER_ERROR, "SERVER_ERROR", Status.INTERNAL_SERVER_ERROR, StandardDownstreamError),
           (Status.SERVICE_UNAVAILABLE, "SERVICE_UNAVAILABLE", Status.INTERNAL_SERVER_ERROR, StandardDownstreamError)
         )
-        input.foreach(args => (serviceErrorTest _).tupled(args))
+
+        val extraTysErrors = Seq(
+          (Status.NOT_FOUND, "NOT_FOUND", Status.NOT_FOUND, NotFoundError),
```

Comment:
Rather than using `Status.xyz` everywhere, instead of ` import play.api.http.Status` can we do `import play.api.http.Status._` ?

Reply:
preference of coding style. I actually prefer to use the Status rather than importing it all. Makes it more explicit that its a status code constant rathe than other constants